testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -61,51 +60,49 @@ def _single_frame(self):
         pass
 
 
-class TestAnalysisBase(TestCase):
-    def setUp(self):
-        # has 98 frames
-        self.u = mda.Universe(PSF, DCD)
-
-    def tearDown(self):
-        del self.u
+class TestAnalysisBase(object):
+    @staticmethod
+    @pytest.fixture()
+    def u():
+        return mda.Universe(PSF, DCD)
 
-    def test_default(self):
-        an = FrameAnalysis(self.u.trajectory).run()
-        assert_equal(an.n_frames, len(self.u.trajectory))
-        assert_equal(an.frames, list(range(len(self.u.trajectory))))
+    def test_default(self, u):
+        an = FrameAnalysis(u.trajectory).run()
+        assert an.n_frames == len(u.trajectory)
+        assert_equal(an.frames, list(range(len(u.trajectory))))
```

Comment:
@kain88-de my understanding is that `assert_equal` is preferred over `assert_array_equal` as discussed in 1444?